### PR TITLE
Update HELP.md

### PR DESCRIPTION
**Pull request by dennydaugherty**
Updating documentation to clarify that headers are indexed for the specific slide trigger.

### CODE REVIEW

**Comment by greyshirtguy** [comment]
Thanks - I love the idea of explaining that headers must be counted!
Can I suggest we simplify the wording?
For example:

**Note:** When indexing presentations in a playlist, you must also include/count any headers.
Example Playlist:

Header A _(index=0)_
Presentation A _(index=1)_
Header B _(index=2)_
Presentation B _(index=3)_

**Comment by dennydaugherty** [comment]
Thanks for the feedback. I've made another commit. Hopefully this will help someone else avoid spending the troubleshooting time it took me to figure this out. :)

**Comment by greyshirtguy** [comment]
Nice, thanks for contributing!
Merged!
